driver-moab/main/cplcomp_exchange_mod.F90

```diff
@@ -1898,7 +1960,9 @@ subroutine cplcomp_moab_Init(infodata,comp)
          call component_exch_moab(comp, mrofid, mbrxid, 0, tagname, context_exch='domr')
          ! copy aream from area in all cases
          ! initialize aream from area; it may have different values in the end, or reset again
-         call copy_aream_from_area(mbrxid)
+         if (mbrxid > 0) then ! on coupler pes only
+            call copy_aream_from_area(mbrxid)
+         endif
 #ifdef MOABDEBUG
          if (mbrxid >= 0) then
```

Comment:
I think all of these checks should verify/assert that app id is strictly above 0. May be in a different PR